src/open_inwoner/utils/tests/test_css.py

```diff
@@ -0,0 +1,53 @@
+from django.test import TestCase
+
+from open_inwoner.utils.css import clean_stylesheet
+
+
+class CSSUtilsTestCase(TestCase):
+    def test_clean_stylesheet_multi(self):
+        tests = [
+            (
+                "* {color: #fff;}",
+                "* {color: #fff;}",
+            ),
+            (
+                "p { color: #fff; }",
+                "p {color: #fff; }",
+            ),
+            (
+                "p {color: #fff; \n width: 0%; \n font-size: 10px;}",
+                "p {color: #fff; \n width: 0%; \n font-size: 10px;}",
+            ),
+            # props not allowed
+            ("p { unknown-prop: 0%;}", ""),
+            (
+                "p {color: #fff; \n unknown-prop: 0%; \n font-size: 10px;}",
+                "p {color: #fff; \n font-size: 10px;}",
+            ),
+            # ignore space and comments
+            ("", ""),
+            (" \n ", ""),
+            ("/* foo */\n \n", ""),
+            ("/* foo */\n \n", ""),
+        ]
+        for i, (css, expected) in enumerate(tests):
+            with self.subTest(i=i):
+                self.assertEqual(expected, clean_stylesheet(css))
+
+    def test_clean_stylesheet_arguments(self):
+        allowed_props = ["width"]
+        css = "body { width:10px; color: #fff;}"
+        expected = "body {width:10px; }"
+
+        self.assertEqual(
+            expected, clean_stylesheet(css, allowed_css_properties=allowed_props)
+        )
+
+    def test_clean_stylesheet_allows_any_string(self):
+        """
+        proofs we need to escape this if we print in html
```

Comment:
Do you mean "proves"? 

It's good to know that we need to do this, hence it should be documented. But what's the point of proving it via a test? I would prefer to have this information included in the docstring of `utils.clean_stylesheet`, where it's more prominent.

Reply:
Yes. 

The problem with comments about code behaviour is they go stale.